crates/rnote-engine/src/pens/penholder.rs

```diff
@@ -253,12 +253,22 @@ impl PenHolder {
             widget_flags |= wf;
         }
 
-        // Always redraw after handling a pen event
+        // Always redraw after handling a pen event.
+        //
+        // This is also needed because pens might have claimed/requested an animation frame.
         widget_flags.redraw = true;
 
         (event_result.propagate, widget_flags)
     }
 
+    /// Handle a requested animation frame.
+    ///
+    /// Can request another frame using `ÈngineViewMut#animation.claim_frame()`.
```

Comment:
Pedantic but the `E` shouldn't have a grave accent

Reply:
fixed, I somehow didn't see that